backend/api.js

```diff
@@ -19,15 +11,37 @@ const foundParams = (params, wantedParam) => {
     return false;
 };
 
-export function v1(requestEvent, sequelizeManager, callback) {
+const obtainDatabaseForTask = (
+    requestEvent, sequelizeManager, sessionSelected, callback
+) => {
+    let database = null;
+    if (foundParams(requestEvent.params, 'database')) {
+        // check if there is a current database in use
+        database = requestEvent.params.database;
```

Comment:
is this comment still accurate? it doesn't seem like we're checking anything here, just assigning `database` to the requested database